### PR TITLE
Feat/profile view

### PR DESCRIPTION
**Pull request by mingard**

### CODE REVIEW

File: frontend/containers/UserProfile/UserProfile.jsx [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+import {buildUrl} from 'lib/router'
+
+import * as userActions from 'actions/userActions'
+
+import SubNavItem from 'components/SubNavItem/SubNavItem'
+
+/**
+ * The interface for editing a user profile.
+ */
+class UserProfile extends Component {
```
Comment (eduardoboucas) [inline frontend/containers/UserProfile/UserProfile.jsx]: Can we do spacing of propTypes consistent with other components?

File: frontend/containers/UserProfile/UserProfile.jsx [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+    } = this.props
+
+
+    if (state.app.config) {
+      const auth = state.app.config.auth
+
+      const currentCollection = getAuthCollection(state.api.apis, auth)
+
+      if (currentCollection) {
+        const firstField = Object.keys(currentCollection.fields).find(field => (currentCollection.fields[field].publish && currentCollection.fields[field].publish.section))
```
Comment (eduardoboucas) [inline frontend/containers/UserProfile/UserProfile.jsx]: Too long? Hasn't the linter complained?

File: frontend/containers/UserProfile/UserProfile.jsx [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+
+  render() {
+    const {sections, section} = this.props
+    const activeSection = section || sections[0].slug
+
+    return (
+      <div class={styles.container}>
+        <div class={styles.navigation}>
+          {sections.map(userSection => {
+            let isActive = userSection.slug === activeSection
```
Comment (eduardoboucas) [inline frontend/containers/UserProfile/UserProfile.jsx]: Could be `const`

File: frontend/containers/UserProfile/UserProfile.jsx (hunk 1 above)
Comment (mingard) [inline frontend/containers/UserProfile/UserProfile.jsx]: Sure. Maybe worth linting

File: frontend/containers/UserProfile/UserProfile.jsx (hunk 2 above)
Comment (mingard) [inline frontend/containers/UserProfile/UserProfile.jsx]: No

Comment (eduardoboucas) [inline frontend/containers/UserProfile/UserProfile.jsx]: We set maximum line length at 80 characters, this is 164.

Comment (mingard) [inline frontend/containers/UserProfile/UserProfile.jsx]: Yes that's right.

File: frontend/containers/UserProfile/UserProfile.jsx (hunk 1 above)
Comment (mingard) [inline frontend/containers/UserProfile/UserProfile.jsx]: Seems to be on the wrong line so this isn't being captured by updates, however it has been resolved

**Pull request merged**